index.html

```diff
@@ -636,10 +636,10 @@ <h3>BitstringStatusListCredential</h3>
 minimum two properties:
                 <ul>
                   <li id="status">
-`status`, being a string of the hex value of the status
+`status`, being a machine-readable string of the hex value of the status
```

Comment:
I think this change isn't necessary? Machines can, of course, read the `status` field. The key thing is that it is a string representation of a hexadecimal value. It's maybe a little curious that you don't specify the format? The examples shows e.g. `0x0` or `0x1`, but `DEADBEEF`, `-1e`, or `10EF` are "hex strings" too. Perhaps:

```suggestion
`status`, a string representing the hexadecimal value of the status prefixed with `0x`
```

Reply:
@mprorock @msporny correct me if i'm wrong, but I believe `0x1` was just an example, and we do not intend to limit hex values to be prefixed with `0x1`.

my understanding of current consensus is that this is a predifined list of status codes that the issuer is responsible for providing a definition for.

Reply:
There is no consensus on this feature outside of the Traceability cohort (other implementers aren't implementing it; we're putting it in there for Transmute, Mesur.io, and Mavennet). 

You guys will need to implement this and tell us how you intended to use this feature.

IIRC, "0x1" wasn't an example, it was intended to express a hexadecimal value in a very specific format. I was presuming that the "message" field contained information that was useful to a developer, but MUST NOT be conveyed to an end-user (which seems to be what we're back to in this PR).

Please check w/ the Traceability folks to help us understand how they want to encode and use the `status` field and the `message` field.

Reply:
> @mprorock @msporny correct me if i'm wrong, but I believe `0x1` was just an example, and we do not intend to limit hex values to be prefixed with `0x1`.
> 
> my understanding of current consensus is that this is a predifined list of status codes that the issuer is responsible for providing a definition for.

we utilize hex codes most often, however, we are open to a definition that does not block use of a hex code of the index - this is NOT user facing information

Reply:
https://github.com/w3ctag/design-principles/issues/453

Reply:
https://github.com/w3ctag/design-principles/issues/454

Reply:
We don't plan to implement this feature.

I suggest cutting everything marked at risk from the spec, simpler for implementers.

Reply:
@OR13 wrote:
> We don't plan to implement this feature.

To be clear, are you stating that Transmute will not be supporting the `"statusPurpose": "message"` mechanism or the "statusMessage" property and its values?

I was under the impression that this feature is required by the Traceability co-hort for the CBP use cases. Is my current understanding incorrect?

Reply:
@OR13 @mkhraisha @mprorock ping again -- trying to get closure on this issue so we can have a clear path to Candidate Recommendation in two weeks.

Reply:
This feature is required by the traceability cohort.
We should not restrict it to hex codes however.

Reply:
Ok, got it, so here's where we are as of right now:

* This is a required feature for Traceability, that's the only place the requirements for this feature are coming from at this moment in time.
* @aphillips seems to agree that if the "message" field is only meant for developers that we don't need a full localization treatment on the field.
* There is no way that we can provide "packed" encoding for the status field unless we use some sort of binary expression for the `status` value.

As such, we're just going to go with all that, merge this PR (with @aphillips additions), and call it a day.

> @mprorock @msporny correct me if i'm wrong, but I believe 0x1 was just an example, and we do not intend to limit hex values to be prefixed with 0x1.

If you want a packed bitfield structure, you're going to have to use a binary representation of some kind. I don't think we can leave this open ended and hope for interop on that requirement from Traceability.

Reply:
@mkhraisha looks like you didn't allow Editors to modify your PR. I'm going to merge what you have and apply @aphillips' changes on top of that.

Reply:
Applied @aphillips changes in 6626331919559b000a71e83845ce49fff0de579b.